### PR TITLE
3-get-api-topics completed

### PR DESCRIPTION
**Pull request by farwah1**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+});
+
+describe('/api/topics', () => {
+    test('GET request responds with an array of topic objects, each of which has slug and description properties', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(( { body }) => {
+            body.forEach(slug => {
+                expect(slug).toMatchObject({
```
Comment (Slyose) [inline __tests__/app.test.js:22]: Also worth testing that the array is of length greater than zero. An empty array has nothing to iterate through with a forEach, it will end up passing even though it's not a response we want.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+const {
+  getTopics
+} = require('./controllers/topics.js');
+
+
+app.get('/api/topics', getTopics);
+
+app.use((err, req, res, next) => {
+  if (err.status) res.status(err.status).send({ msg: err.msg });
```
Comment (Slyose) [inline app.js:13]: This error handler is good, however it's not relevant at this stage in the project. As the code is right now, this will never be called. Worth taking out and re-introducing later on to a relevant ticket. Try not to over-engineer too early, instead focus on the ticket itself.

**Review by Slyose — APPROVED** [review]
Feel free to merge after taking on feedback!
